### PR TITLE
Describe configuration pushes and webhooks.

### PR DESCRIPTION
**Pull request by gluszczyk**
agents
bots
auto accesses

## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-8724-config-events)
- [Jira](https://livechatinc.atlassian.net/browse/API-8724)

## 📓 Description

[agents ticket](https://livechatinc.atlassian.net/browse/API-8714)
[auto access ticket](https://livechatinc.atlassian.net/browse/API-8733)

I'll fix the formatting once we settle on the wording 😉 

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes

### CODE REVIEW

File: content/management/changelog/index.mdx [hunk 1]
```diff
@@ -24,6 +24,19 @@ The developer preview version provides a preview of the upcoming changes to the
 ### Webhooks
 
 - The [`source_type`](/management/configuration-api/v3.4/#register-webhook) filter was added to all webhooks.
+- There are new configuration webhooks that can be used to track changes to license resources:
```
[the remark this answers is not in the document]
Comment (gluszczyk) [reply content/management/changelog/index.mdx:27]: yeah, good idea, both PRs touch parts of the same changeset, so whichever introduces such description, the other should keep it

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 2]
```diff
@@ -23,7 +23,25 @@ The developer preview version provides a preview of the upcoming changes to the
   - [**chat_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#chat_deleted)
   - [**thread_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#thread_deleted)
   - [**threads_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#threads_deleted)
-  - [**configuration-related pushes**](/messaging/agent-chat-api/v3.4/rtm-reference/#configuration)
+
+### Configuration
+
+- Updating license resources in [the Configuration API v3](/management/configuration-api/) now results in pushes informing agents about applied changes. These pushes are as follows:
```
Comment (bkubiak) [inline content/messaging/agent-chat-api/changelog/index.mdx:29]: perhaps, we should emphasise here that it concerns all version: v3.1, v3.2, v3.3, v3.4 - I'm not sure if `the Configuration API v3` is enough

**Review by bkubiak — APPROVED** [review]
Great work!

File: content/messaging/agent-chat-api/changelog/index.mdx (hunk 2 above)
Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx:29]: Perhaps v3.x would solve the problem? We already use the term in the Changelog 
image

Comment (bkubiak) [reply content/messaging/agent-chat-api/changelog/index.mdx:29]: ok

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**